chapters/Chapter 2 — Dressara.md

```diff
@@ -0,0 +1,94 @@
+The women of Dressara often wore dresses, which Leanne both admired and found a shame. Leanne enjoyed wearing dresses, but found women in suits much more handsome.
+
+Delphine was nothing if not handsome. She was tall, and did her best to look so: her fine Elisian suit lengthened her torso, and she had paired it, rather than with pants, with a skirt just long enough to wave gently in the light breeze the Dressarans were so fond of.
+
+Dressara was a large moon. It was somewhat pleasant, if you enjoyed their sort of pomp, unlike the dour planet Seven around which it held a lazy orbit. Like anywhere else, it was rather difficult to land on illegally, particularly when it was in the process of adjusting its orbit. 
+
+It was simpler to land legally. While Leanne didn’t have an Alt to identify her, nor to pilot her ship, she did have a collection of bounty cards for people on all the various planets and moons. Their targets were now long dead, but so long as the bodies weren’t found, Leanne could use their bounties to land (bounties did not expire).
+
+There only a few thousand inhabitants on Dressara, plus their few thousand servants. There was a party nearly every day, and everyone was invited—sometimes the servants, too. The men, Leanne knew, did little more than stand about all day, whether at parties or at home. They were utterly boring, both as people and as targets. The women, Leanne, assumed, did something more than simply wear dresses and stroll about—even the wealthy had to bring in money somehow—but whatever they did, it wasn’t interesting enough for Leanne to learn what it was. 
```

Comment:
> There only a few thousand inhabitants

"Dressara held only a few thousand inhabitants, plus their few thousand servants."

"Dressara was home only to a few thousand people, plus their few thousand servants." That's perhaps a bit too strong on the "treating servants like they aren't people" train of thought, particularly as it isn't really Leanne's point of view (more her observation of Dressarans' points-of-view)

"Dressara was home to only a few thousand, plus their few thousand servants. There were parties nearly every day, and everyone was invited—sometimes the servants, too. The men, Leanne knew, did little more than stand about all day, whether at home or at parties. They were utterly boring, both as targets and human beings. The women, Leanne assumed, did something more than simply wear dresses and stroll about—even the wealthy had to rake in more money somehow—but whatever they did, it was never interesting enough for Leanne to learn what it was."